### DISCUSSION TITLE
TrajOptTaskflow Failure

### DISCUSSION THREAD

**Discussion by vegfood** [tesseract-robotics/tesseract_planning#238] [Q&A] [answered]
when I use trajopt planner, sometimes it fails and shows that  <<Not all constraints are satisfied>> even if the plan problem seems not difficult. I also looked at the plotted trajectory in rviz, it seems all right and no collision, so i wander why it reports failed message and what exactly are the constraints taken into account while planning, i  know there're pose, start and end position, what else ?

**Comment by Levi-Armstrong** [comment, marked as answer]
Sorry, I missed this post. The easiest way to determine why it failed is to set the logger level to debug which will cause it to print out a summary table of the costs and constraints. This will help identify where the issue is.

**Reply by vegfood** [reply]
ok, thank you ！
